### DISCUSSION TITLE
Werkzeug Zeichnen: Hochladen von GPX Dateien - Waypoints werden nicht angezeigt

### DISCUSSION THREAD

**Discussion by fanie-mak** [e-netze/webgis-community#100] [General]
Wenn ich eine GPX Datei hochlade, werden die Waypoints leider nicht angezeigt. In der Karte wird jedoch auf den richtigen Bereich gezoomt.
Dieses Verhalten tritt auf, wenn ich eine GPX-Datei im WebGIS erstellt habe und diese dann hochlade, aber auch mit GPX-Dateien, die von einer anderen Quelle heruntergeladen worden sind und dann im WebGIS hochgeladen werden. 

Tracks funktionieren einwandfrei...

**Comment by gis-eni** [comment]
Bug: wird mit Build 5.23.3601 behoben
